Sources/Tropos/Views/CircularProgressLayer.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+@objc(TRCircularProgressLayer) class CircularProgressLayer: CALayer {
+    @objc var progress: CGFloat = CGFloat(0)
+    @objc var radius = CGFloat(15.0)
+    @objc let outerRingWidth = CGFloat(3.0)
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override init() {
+        super.init()
+        self.actions = [
+            "bounds": NSNull(),
```

Comment:
Curious, does a `nil` literal work here? AFAIK `nil` _should_ be bridged to `NSNull`, but I’m not sure if that works for `CAAction`.